nf_core/modules/lint/module_tests.py

```diff
@@ -34,8 +34,7 @@ def module_tests(module_lint_object, module):
             else:
                 module.failed.append(("test_pytest_yml", "missing entry in pytest_software.yml", pytest_yml_path))
     except FileNotFoundError as e:
-        log.error(f"Could not open pytest_software.yml file: {e}")
-        sys.exit(1)
+        module.failed.append((f"Could not open pytest_software.yml file: {e}"), pytest_yml_path)
```

Comment:
This doesn't seem right, you can only append tuples of `("test_name", "message", "path_to_file)` to  the `module.failed` list (or the other test lists), otherwise it will cause an error down the line. So this should work:

```suggestion
        module.failed.append(("test_pytest_yml", f"Could not open pytest_software.yml file", pytest_yml_path))
```

But would be good to check of course :)

Reply:
Thanks! I'll test it out a bit then.